stable/artifactory-ha/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [0.16.0] - Aug 21, 2019
```

Comment:
@eldada Don't you think its time to go with version 1.0?

Reply:
Probably true, but I want to move to the statefuleset postgresql first.